relay/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./*"],
+      "@/lodestar-types": ["../.yarn/unplugged/@lodestar-types-npm-1.17.0-1607a25762/node_modules/@lodestar/types/lib/utils/executionAddress"]
```

Comment:
re: `@lodestar-types-npm-1.17.0-1607a25762`

Isn't it a problem that we're hardcoding a version here?

Reply:
We are unplugging specific version:
```json
    "@lodestar/types@1.17.0": {
      "unplugged": true
      }
```

Or do you refer to the hash after the version?

Reply:
No, I was referring to 1.17.0. Okay, I guess... So what happens when lodestar upgrades?

Reply:
This is a valid question. I guess we have to re-unplug? And update the path here. Hmm..